packages/build/bin/run-clean.js

```diff
@@ -24,7 +24,7 @@ Example usage:
 'use strict';
 
 function run(argv, options) {
-  const fs = require('fs-extra');
+  const rimraf = require('rimraf');
   const path = require('path');
   const utils = require('./utils');
   var files = argv.slice(2);
```

Comment:
Please rename `files` to something like `patterns` to make it more obvious that we are accepting wildcards.